apps/web/app/(ee)/admin.dub.co/(dashboard)/components/user-info.tsx

```
@@ -53,30 +60,16 @@ export default function UserInfo({ data }: { data: UserInfoProps }) {
-              <span className="text-neutral-500">
-                {nFormatter(workspace.clicks, { full: true })}
-              </span>
-            </div>
-            <div className="flex justify-between text-sm">
-              <span className="font-medium text-neutral-700">Sales</span>
-              <span className="text-neutral-500">
-                {currencyFormatter(workspace.sales / 100)}
-              </span>
-            </div>
+            {items.map((item) => (
+              <div className="flex justify-between text-sm">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)